run-server 2.sh

```
@@ -0,0 +1,31 @@
+
+# Check if the dist directory exists, if not, build the project
+if [ ! -d "./dist" ]; then
+    echo "Dist directory not found. Building project..."
+    npm run build
+fi
+
+# Check if OCTAGON_API_KEY is set
+if [ -z "$OCTAGON_API_KEY" ]; then
+    # Try to load from .env file
+    if [ -f ".env" ]; then
+        export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)